worker/utils/utils.js

```diff
@@ -7,6 +7,9 @@ const yaml = require('js-yaml');
 const { promisify } = require('util');
 const exec = promisify(require('child_process').exec);
 const mongo = require('./mongo');
+const DB_NAME = process.env.DB_NAME ? process.env.DB_NAME : 'pool';
+var crypto = require('crypto');
```

Comment:
const DB_NAME is created here but not referenced, you can remove it. Instead of var, it's fine to use const for the crypto import. Using elint can catch some of these for you. https://eslint.org/docs/user-guide/getting-started. I use the air B&B style guide, and just have it locally installed for now (run the commands in the tutorial from worker dir).